presto-docs/src/main/sphinx/release/release-335.rst

```diff
@@ -0,0 +1,42 @@
+=========================
+Release 335 (XX May 2020)
+=========================
+
+General Changes
+---------------
+
+* Fix failure when :func:`reduce_agg` is used as a window function. (:issue:`3883`)
+* Fix incorrect cast from ``TIMESTAMP WITHOUT TIME ZONE`` to ``TIME`` type. (:issue:`3848`)
+* Improve optimized local scheduling with regard to non-uniform data distribution. (:issue:`3922`)
+* Add support for variable-precision ``TIMESTAMP WITHOUT TIME ZONE`` type. (:issue:`3783`)
+* Return ``VARCHAR`` type when :func:`substr` argument is ``CHAR`` type. (:issue:`3599`)
+* Add a variant of :func:``substring`` that takes a ``CHAR`` argument. (:issue:`3949`)
+
+Security Changes
+----------------
+
+* Add  ``information_schema.role_authorization_descriptors`` table that returns information about what roles where granted to what principals. (:issue:`3535`)
+* Adding schema access rules to file based system access control. (:issue:`3766`)
+
+Hive Connector Changes
+----------------------
+
+* Fix a bug that caused missing column statistics after ``INSERT`` to new table. (:issue:`2469`)
+* Add support for ``information_schema.role_authorization_descriptors`` table when ``hive.security=sql-standard``.  (:issue:`3535`)
+* Allow non-lowercase column names in ``system.sync_partition_metadata`` procedure. This can be enabled
+  by passing ``case_sensitive=false`` when invoking the procedure. (:issue:`3431`)
+* Make caching work when coordinator is secured. (:issue:`3874`)
+* Prevent caching from turning off in case of intermittent network failures. (:issue:`3874`)
+* Make caching and HDFS impersonation mutually exclusive. (:issue:`3913`)
```

Comment:
Maybe change to

Ensure HDFS impersonation and caching are mutually exclusive.

Cant think of a better way atm. Make mutually exclusive .. sounds weird to me.

Reply:
@sopel39, can you clarify what this is trying to convey?

Reply:
> Ensure HDFS impersonation and caching are mutually exclusive.

That sounds better. Thank you.

Basically both impersonation and caching cannot be on at the same time

Reply:
Maybe
```
Ensure HDFS impersonation is not enabled when caching is enabled. Previously it 
was possible to enable HDFS impersonation when using caching, but the impersonation
was ineffective.
```

?

Reply:
Thats too long. We could have that info in the docs somewhere but since it would just document a broken setup I dont think we should bother.